Content.Shared/Language/LanguagePrototype.cs

```diff
@@ -0,0 +1,37 @@
+using System.Runtime.CompilerServices;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Language;
+
+[Prototype("language")]
+public sealed class LanguagePrototype : IPrototype
+{
+    [IdDataField]
+    public string ID { get; private set;  } = default!;
+
+    // <summary>
+    // If true, obfuscated phrases of creatures speaking this language will have their syllables replaced with "replacement" syllables.
+    // Otherwise entire sentences will be replaced.
+    // </summary>
```

Comment:
```suggestion
    /// <summary>
    ///     If true, obfuscated phrases of creatures speaking this language will have their syllables replaced with "replacement" syllables.
    ///     Otherwise entire sentences will be replaced.
    /// </summary>
```